sips/sip-021/sip-021-nakamoto.md

```diff
@@ -0,0 +1,542 @@
+# Preamble
+
+SIP Number: 021
+
+Title: Fast and Reliable Blocks through PoX-assisted Block Propagation
+
+Authors:
+
+* Aaron Blankstein <aaron@hiro.so>
+* Charlie Cantoni <charlie@hiro.so>
+* Brice Dobry <brice@hiro.so>
+* Jacinta Ferrent <jacinta@trustmachines.co>
+* Diwaker Gupta <diwaker@hiro.so>
+* Jesus Najera <jnajera1917@gmail.com>
+* Jude Nelson <jude@stacks.org>
+* Ashton Stephens <ashton@trustmachines.co>
+* Joey Yandle <joey@trustmachines.co>
+
+Consideration: Governance, Technical
+
+Type: Consensus
+
+Status: Draft
+
+Created: 2023-09-28
+
+License: BSD 2-Clause
+
+# Abstract
+This document describes a consensus-breaking change to the Stacks blockchain that would enable faster and more reliable Stacks blocks.
+
+In this proposal, Stacks block production would no longer be tied to miner elections. Instead, miners produce blocks at a fixed cadence, and the set of PoX Stackers rely on the miner elections to determine when the current miner should stop producing blocks and a new miner should start. This blockchain will only fork if 70% of Stackers approve the fork, and chain reorganization will be as difficult as reorganizing Bitcoin.
+
+This proposal, dubbed the "Nakamoto" release, represents a substantial architectural change to the current Stacks blockchain. If adopted, the Stacks major version would be bumped from 2 to 3. The first Nakamoto release would be 3.0.0.0.0.
+
+# Introduction
+
+## Glossary
+
+### Existing Terminology
+|Term|Definition|
+|-|-|
+|burnchain|The chain that a stacks blockchain follows in the PoX protocol. The prefix “burn” is a vestige of  [SIP-001][SIP-001-link]’s terminology where a fungible token would be burned on a chain to determine a miner’s chance of being elected leader / miner of the next block. After the activation of  [SIP-007][SIP-007-link], burnchain tokens are no longer burned; they are transferred to stackers. <br><br> The most popular and relevant instance of the Stacks blockchain utilizes Bitcoin as its burnchain. Where previous SIPs have defined behavior relative to a nameless burnchain, this SIP will simply assume that the burnchain in use is Bitcoin for ease of reading.|
+|cryptographic sortition|A process of randomly selecting one or more entities from a set using cryptography. This is a decentralized and verifiable way to select participants for a variety of tasks, such as consensus protocols, lotteries, and auctions.|
+|miner selection (aka. sortition)|Also known as “leader selection,” “miner election” or simply “sortition,” miner selection is the weighted cryptographic sortition process by which a miner candidate is selected as the next miner (leader). Details of this process are in [SIP-001][SIP-001-link] with mechanism alterations in [SIP-007][SIP-007-link].<br><br>Within this SIP, we will refer to the miner selection process as simply “the sortition.”|
```

Comment:
```suggestion
|miner selection (aka. sortition)|Also known as “leader selection”, “miner election” or simply “sortition”, miner selection is the weighted cryptographic sortition process by which a miner candidate is selected as the next miner (leader). Details of this process are in [SIP-001][SIP-001-link] with mechanism alterations in [SIP-007][SIP-007-link].<br><br>Within this SIP, we will refer to the miner selection process as simply “the sortition".|
```

Reply:
I believe this is just American (as-is) vs British (your suggestion) standards. Either could be correct for this doc.

Reply:
I should admit... never realized that!!!  Thanks @obycode Now I am seeing `.,"` everywhere! 👀

Reply:
@csgui and I were just chatting about this, and the lack of an Oxford comma in this sentence 😱 😄. At some point, it would be awesome if there was a team of technical writers just focused on this part of these docs, at which point we could pick some common style guide and stick with it. A tech writer I worked with previously liked Microsoft's style guide - https://learn.microsoft.com/en-us/style-guide/

Regardless, this is not an urgent issue at the moment. Clarity is the only requirement for now.